### DISCUSSION TITLE
Ran into trouble setting up basic notebook on Windows 10 and WSL2

### DISCUSSION THREAD

**Discussion by fazil47** [nteract/archived-desktop-app#5320] [Q&A]
This is my App.js (This is the only file I edited after using npx create-react-app)
![image](https://user-images.githubusercontent.com/18116695/95670337-5f39ce80-0ba7-11eb-9b11-b69a6e80c4d2.png)
And this is the output
![image](https://user-images.githubusercontent.com/18116695/95670342-75478f00-0ba7-11eb-8750-2b429c363c71.png)

**Comment by captainsafia** [comment]
Thanks for opening up this thread, @fazil47!

It looks like you're looking to get started with our core SDK and have use the create-nteract-app command. Unfortunately, that utility hasn't been updated in a while and generates a bad scaffold. 

To get started with this, I'd recommend looking at the same implementation in the nteract desktop app at https://github.com/nteract/nteract/blob/1a674f787bb68405bd508f06be524dc4a72f08da/applications/desktop/src/notebook/index.tsx.

We're planning on improving our SDK docs, but in the meantime I'm happy to continue answering questions you have on this thread.

**Reply by fazil47** [reply]
I'll try that, thank you :)
